docs/manual-test.md

```diff
@@ -232,3 +239,16 @@ Create a new workbook on Tableau Server with the connector installed on the serv
 
     **Note:** Web authoring (creating a new connection from the web) is not currently available for all connector superclasses. In those cases, your connector won't appear on the list of connectors on Tableau Server. If you can publish a workbook or data source using your connector to your server, then your connector is loaded correctly, even if you can't see it on the list of connectors.
 
+__Test refresing OAuth token on Tableau Server__
+
+If your connector supports oauth, you need to perform this extra step to make sure Tableau can successfully refresh your token.
+1. Go to user's server settings page and find the pane for Saved Credentials for Data Sources.
+
+1. Find your connector in the connector list and click **Add** button next to it.
+
+1. This will invoke the OAuth flow, go ahead and authenticate and this will save the oauth token securly in Tableau Server.
+
+1. Exam the oauth token saved, it should contain a username which uniquely identify you, it may also contain a instanceUrl if your oauthConfig file has OAUTH_SUPPORTS_CUSTOM_DOMAIN enabled.
```

Comment:
```suggestion
1. Examine the OAuth token saved, it should contain a username which uniquely identify you. It may also contain a instanceUrl if your oauthConfig file has OAUTH_SUPPORTS_CUSTOM_DOMAIN enabled.
```